core/src/main/java/nu/yona/server/subscriptions/entities/NewDeviceRequest.java

```diff
@@ -0,0 +1,83 @@
+package nu.yona.server.subscriptions.entities;
+
+import java.util.Date;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import nu.yona.server.crypto.CryptoSession;
+import nu.yona.server.crypto.StringFieldEncrypter;
+import nu.yona.server.entities.EntityWithID;
+import nu.yona.server.entities.RepositoryProvider;
+
+/*
+ * A request to add another device for an existing user. The data cannot be encrypted with the user 'password' or auto-generated
+ * key because that is stored on the device and cannot be entered by the user. Therefore we have to transfer the user 'password'
+ * to the new device and obviously encrypt it (with a secret entered by the user, e.g. her PIN or a one time password) while
+ * transferring.
+ */
+@Entity
+@Table(name = "NEW_DEVICE_REQUESTS")
+public class NewDeviceRequest extends EntityWithID
+{
+	public static NewDeviceRequestRepository getRepository()
+	{
+		return (NewDeviceRequestRepository) RepositoryProvider.getRepository(NewDeviceRequest.class, UUID.class);
+	}
+
+	private Date expirationDateTime;
```

Comment:
Please make this a ``creationDateTime``. That makes changing the cleanup policy more flexible and provides more information when debugging the system.